tests/test_traceview.py

```diff
@@ -134,6 +137,22 @@ def test_error_rates(self):
         self.assertIsInstance(error_rates, dict)
         self.assertTrue('items' in error_rates)
 
+@unittest.skipIf(TV_API_KEY is None, "No TraceView API Key found in environment.")
+@unittest.skipIf(TV_REQUESTS_APP is None, "TV_REQUESTS_APP must define a valid (non-Default) app in order to test the total_requests API.")
+class TestTotalRequests(unittest.TestCase):
+
+    def setUp(self):
+        self.tv = traceview.TraceView(TV_API_KEY)
+
+    def test_error_rates(self):
+        apps = self.tv.apps()
+        self.assertTrue(len(apps) > 0)
+
+        total_requests = self.tv.total_requests(TV_REQUESTS_APP)
+        self.assertNotEqual(total_requests, None)
+        self.assertIsInstance(total_requests, dict)
+        self.assertTrue('items' in total_requests)
```

Comment:
@mcodd this can be updated to use the `self.assertIn` method instead of `assertTrue` (i.e. `self.assertIn('items', total_requests)`)

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIn